### PR TITLE
Update from node-template

### PR DESCRIPTION
**Pull request by BenGitsCode**
Refresh for cohort 016

update package.json

### CODE REVIEW

File: lib/example.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
```
Comment (jrhorn424) [inline lib/example.js]: This file is not needed.

File: package.json [hunk 2]
```diff
@@ -2,20 +2,26 @@
   "name": "ga-wdi-boston.js-reference-types-practice",
   "version": "0.0.1",
   "private": true,
-  "license": "MIT",
+  "scripts":{
+    "start": "node bin/example.js"
```
Comment (jrhorn424) [inline package.json]: This `scripts` property isn't need for now.

**Review by jrhorn424 — CHANGES_REQUESTED** [review]
Thanks! Please address comments when you get a moment.

Then we'll need to update the solution branch and bonus branch.

**Comment by BenGitsCode** [comment]
Made those changes Jeff, I made the mistake of pushing before amending my commit, as I'm sure you'll notice.

**Comment by payne-chris-r** [comment]
Can we close or resolve this?

**Comment by payne-chris-r** [comment]
@BenGitsCode @jrhorn424 ^

**Comment by BenGitsCode** [comment]
@payne-chris-r I think this can be closed if Jeff's okay with that. You're working on the lm01 update?

**Comment by payne-chris-r** [comment]
Thanks @BenGitsCode

**Pull request closed without merging**